contents/docs/data/index.mdx

```diff
@@ -6,172 +6,108 @@ availability:
     free: partial
     selfServe: full
     enterprise: full
-
-    features:
-        tagging:
-            free: false
-            selfServe: true
-            enterprise: true
-        descriptions:
-            free: false
-            selfServe: true
-            enterprise: true
 ---
 
-[//]: # 'On restricted plans, users cannot create definitions and view any 30d metrics for events and event properties.'
-
-Data management lets you keep events and properties organized. Managing your data becomes especially helpful in situations where:
-
--   It is hard to understand events, actions, and properties that you yourself didn't set up.
--   Data decays quickly, and most of the time fails to stay up-to-date with instrumentation.
--   Event-level context in insights and dashboards is difficult to share with external people and new team members.
-
-## What is a definition?
-
-Definitions are the basic building blocks for how you manage data at PostHog. You don't need to create definitions in order to get started analyzing your data on PostHog, but your team's experience on PostHog can be greatly enriched when your events are defined with best practices in mind.
-
-It is also important to note that PostHog will implicitly create a skeleton definition for every event and event property it ingests, and each definition
-can be enriched with more context at any time.
-
-## When do I need a definition?
+Data management lists many of the types of data in your PostHog instance. This helps you better understand the data you have and how it is used.
```

Comment:
We're already in the docs, so I dont think we need to say "this helps you better..."
```suggestion
Below are the different types of data you'll come across in your PostHog instance. 
```